orca/templates/db/resources/etc/mysql/mysql.conf.d/mysqld.cnf

```diff
@@ -0,0 +1,7 @@
+[mysqld]
```

Comment:
Let's move things below in ABC order of possible.

Reply:
Done